scripts/zfs.sh

```diff
@@ -110,10 +111,11 @@ check_modules_linux() {
 load_module_linux() {
 	KMOD=$1
 
+	FILE=$(modinfo "$KMOD" | awk '/^filename:/ {print $2}')
+	[ "$FILE" = "(builtin)" ] && return
```

Comment:
This generates a warning for me on RHEL/CentOS 8 which used to be suppressed.  Presumably it's due to the older `modinfo` version which doesn't understand builtin modules.

```sh
$ ./scripts/zfs.sh 
modinfo: ERROR: Module zlib_deflate not found.

Failed to load zlib_deflate

$ modinfo zlib_deflate
modinfo: ERROR: Module zlib_deflate not found.
```

```sh
$ modinfo --version
kmod version 25
+XZ +ZLIB +OPENSSL -EXPERIMENTAL
```

Reply:
Special(ish) handling for zlib modules restored, is now
```sh
FILE=$(modinfo "$KMOD" 2>&1 | awk 'NR == 1 && /zlib/ && /not found/ {print "(builtin)"; exit}  /^filename:/ {print $2}')
```

I guess this does also suppress the warning for a missing zfs.ko but we unconditionally load it below, so we avoid a doubled error message.